src/baseline.c

```diff
@@ -501,7 +501,7 @@ s8 lesq_solve_raim(u8 num_dds_u8, const double *dd_obs,
     if (n_used) {
       *n_used = 0;
     }
-    return -3;
+    return -5;
```

Comment:
We really do need to fix this return, is it silly to have to returns with the same value.

I plan to make another PR that adds error message strings for these codes, similar to `calc_PVT`